pkg/bitwarden/cli.go

```diff
@@ -142,3 +151,246 @@ func (c *cliClient) GetPassword(itemName string) ([]byte, error) {
+	if _, err := c.runWithSession("delete", "attachment", attachmentID, "--itemid", itemID); err != nil {
+		return fmt.Errorf("failed to delete attachment, attachmentID: %s, itemID: %s: %w", attachmentID, itemID, err)
+	}
+	return nil
+}
+
+func (c *cliClient) SetFieldOnItem(itemName, fieldName string, fieldValue []byte) error {
+	var targetItem *Item
+	var targetField *Field
+	for index, item := range c.savedItems {
+		if itemName == item.Name {
+			targetItem = &c.savedItems[index]
+			for fieldIndex, field := range item.Fields {
+				if field.Name == fieldName {
+					targetField = &c.savedItems[index].Fields[fieldIndex]
+					break
+				}
+			}
+			break
+		}
+	}
+	if targetItem == nil {
+		newItem := &Item{}
+		if err := c.createEmptyItem(itemName, newItem); err != nil {
+			return fmt.Errorf("failed to create new	 bw entry: %w", err)
+		}
+		c.savedItems = append(c.savedItems, *newItem)
+		targetItem = &c.savedItems[len(c.savedItems)-1]
+	}
+	isNewField := false
+	if targetField == nil {
+		targetItem.Fields = append(targetItem.Fields, Field{fieldName, string(fieldValue)})
+		targetField = &targetItem.Fields[len(targetItem.Fields)-1]
+		isNewField = true
+	}
+	if isNewField || targetField.Value != string(fieldValue) {
+		targetField.Value = string(fieldValue)
+		if err := c.editItem(*targetItem); err != nil {
+			return fmt.Errorf("failed to set field, itemName: %s, fieldName: %s - %w", itemName, fieldName, err)
+		}
+	}
+	return nil
+}
+
+func (c *cliClient) SetAttachmentOnItem(itemName, attachmentName string, fileContents []byte) error {
+	var targetItem *Item
+	var targetAttachment *Attachment
+	var targetAttachmentIndex int
+	for index, item := range c.savedItems {
+		if itemName != item.Name {
+			continue
+		}
+		targetItem = &c.savedItems[index]
+		for attachmentIndex, attachment := range item.Attachments {
+			if attachment.FileName == attachmentName {
+				targetAttachmentIndex = attachmentIndex
+				targetAttachment = &c.savedItems[index].Attachments[attachmentIndex]
+				break
+			}
+		}
+		break
+
+	}
+	if targetItem == nil {
+		newItem := &Item{}
+		if err := c.createEmptyItem(itemName, newItem); err != nil {
+			return fmt.Errorf("failed to create new	 bw entry: %w", err)
+		}
+		c.savedItems = append(c.savedItems, *newItem)
+		targetItem = &c.savedItems[len(c.savedItems)-1]
+	}
+	attachmentChanged := true
+	if targetAttachment != nil {
+		// read the attachment file
+		tempDir, err := ioutil.TempDir("", "attachment")
+		if err != nil {
+			return fmt.Errorf("failed to create temporary file for getting: %w", err)
+		}
+		defer os.RemoveAll(tempDir)
+		filePath := filepath.Join(tempDir, attachmentName)
+		existingFileContents, err := c.getAttachmentOnItemToFile(itemName, attachmentName, filePath)
+		if err != nil {
+			return fmt.Errorf("error reading attachment: %w", err)
+		}
+		if bytes.Equal(fileContents, existingFileContents) {
+			attachmentChanged = false
```

Comment:
Just `return nil` here to save yourself the variable, the else tree and the `if attachmentChanged`